src/applications/vaos/appointment-list/pages/PastAppointmentsPage/index.jsx

```diff
@@ -63,15 +62,13 @@ export default function PastAppointmentsPage() {
   }, []);
   useEffect(
     () => {
-      if (pastStatus === FETCH_STATUS.succeeded && !isInitialMount) {
-        scrollAndFocus('h3');
-      } else if (hasTypeChanged && pastStatus === FETCH_STATUS.succeeded) {
-        scrollAndFocus('#type-dropdown');
-      } else if (hasTypeChanged && pastStatus === FETCH_STATUS.failed) {
+      if (pastStatus === FETCH_STATUS.succeeded) {
```

Comment:
The current production behavior of the focus is on the h1 upon initial visit to the Past appointment page and remains on h1 even after the default 3 months appointments finish loading.  

I think the request for change is to make focus from h3 (appointment month) to “Showing x appointments between today and [date]”  after the initial mount. 
```suggestion
      if (pastStatus === FETCH_STATUS.succeeded && !isInitialMount) {
```

Reply:
I suspected that the existing behavior has been obsolete for a while now since 

```
} else if (hasTypeChanged && pastStatus === FETCH_STATUS.succeeded) {
        scrollAndFocus('#type-dropdown');
```

Suggested the original intention was to focus on the drop down on initial load, then focus on `h3` on subsequent loads. However, the id `type-dropdown` doesn't seem to exist and when I was testing it, regardless of if it was the initial load or not, it never focused on the drop down. Since this logic was old I wasn't sure if there was a reason to keep the focus on `h1` for the initial load succeeds. 

I'm not sure if we want to preserve the existing behavior since the focus is on the `h1` (i.e. "Past appointments") while the appointments are loading and once it loads, I think it makes more sense to change the focus to `#appointment-count` (i.e. "Showing X appointments between today and Y"). @outerpress what do you think? Should we keep the focus on "Past appointments" for the first load (e.g. the default fetch for 3 months) then change to the appointment count text on subsequent loads?

Reply:
> I think it makes more sense to change the focus to #appointment-count (i.e. "Showing X appointments between today and Y"). @outerpress what do you think? Should we keep the focus on "Past appointments" for the first load (e.g. the default fetch for 3 months) then change to the appointment count text on subsequent loads?

Yep! When users initially land on the page we don't want focus to start on the count. That H1 gives them context as to where they are. 

We should only manage focus to land on the count when they're actively using the date range filter dropdown. The experience should be:

1. I select a date range
1. If the page load indicator starts, I hear the page is loading some data 
2. I hear the count of what loaded